tests/Functional/Schema/SchemaManagerTest.php

```diff
@@ -275,4 +275,63 @@ public function testIntrospectTableWithInvalidName(): void
 
         $this->schemaManager->introspectTable('"example');
     }
+
+    public function testIntrospectTableWithDotInIndexNames(): void
+    {
+        $table = Table::editor()
+            ->setQuotedName('test_user')
```

Comment:
> I don't know why setting the FK on DB2 and Oracle DBs fails, happy for any insights how to fix

You declare the table name here as quoted but reference it in the other table as unquoted: https://github.com/doctrine/dbal/blob/d7ff279deea6c631047c83007b0c6db82cf194b6/tests/Functional/Schema/SchemaManagerTest.php#L318

On Oracle and Db2, `"test_user"` (quoted) remains `test_user`, while  `test_user` (unquoted) becomes `TEST_USER`. Hence the error message:

> "DB2INST1.TEST_USER" is an undefined name